src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Produces_UnparsableContentType" xml:space="preserve">
+    <value>The argument '{0}' could not be parsed. See the inner exception for more information.</value>
+  </data>
+  <data name="Produces_MatchAllContentType" xml:space="preserve">
+    <value>The argument '{0}' is invalid. A match all value for the media-type and media-subtype is not allowed. Consider providing a more specific content type.</value>
```

Comment:
`A match all value for the media-type and media-subtype is not allowed.`

A bit misleading - this implies that a wildcard can be used for one *or* the other but not both. 

I'd just say

> `MediaTypes containing wildcards (*) are not supported.


Also drop the `Consider providing a more specific content type.` - it's not a suggestion :laughing: